CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Ewon Flexy Extensions Library Changelog
```

Comment:
Release commit is missing required changes. See a previous release commit, such as https://github.com/hms-networks/sc-ewon-flexy-extensions-lib/commit/550a530a65d67cfa96c51355b29d612dcff9b4d8.